marimo/_plugins/ui/_impl/dataframes/dataframe.py

```diff
@@ -125,6 +129,8 @@ def __init__(
         page_size: Optional[int] = 5,
         limit: Optional[int] = None,
         show_download: bool = True,
+        download_csv_encoding: Optional[str] = "utf-8",
```

Comment:
I believe this shouldn't be optional, since `None` is not a valid value (I assume). (Having a default value is different from being optional)